telegram/_bot.py

```diff
@@ -3661,7 +3661,7 @@ async def get_updates(
             await self._post(
                 "getUpdates",
                 data,
-                read_timeout=read_timeout + timeout if timeout else read_timeout,
+                read_timeout=read_timeout + float(timeout) if timeout else read_timeout,
```

Comment:
float + integer should work without explicit conversion, shoudn't it? 🤔

Reply:
I don't know, actually 😅 I'm coming from Swift, and I thought it doesn't work like that. So I'll revert this change 👍🏻